packages/style-engine/class-wp-style-engine.php

```diff
@@ -365,7 +365,7 @@ public static function process_and_enqueue_stored_styles() {
 		foreach ( $stores as $key => $store ) {
 			$processor = new WP_Style_Engine_Processor();
 			$processor->add_store( $store );
-			$styles = $processor->get_css();
+			$styles = $processor->get_css( array( 'prettify' => true ) );
```

Comment:
We should probably not default to beautified styles... They should only be enabled if `SCRIPT_DEBUG` is `true` 👍 
```suggestion
			$styles = $processor->get_css( array( 'prettify' => defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) );
```

Reply:
That's weird. I wanted it off by default. I must have pushed the wrong thing. Thanks for flagging.